docs/mux-logging-service.md

```diff
@@ -49,6 +49,20 @@ There are environment variables for the logging-mux DC config:
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
+These are environment variables for the logging-fluentd daemonset config:
+
+* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
+  will not send logs directly to Elasticsearch, instead it will use
+  `secure_forward` to send logs to mux.
+* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset - if
+  `minimal`, this is the same behavior as `USE_MUX_CLIENT true`.  If
+  `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
```

Comment:
Can we consider not giving these options at all and just always deploying using what is essentially `full_no_k8s_meta`?  Would that simplify the logic necessary to deploy all this?  If we only deployed logging one way that scales, I am not sure we would need to support the old way at all.